samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -367,6 +368,193 @@ To monitor the GNSS module (for instance, to see whether A-GPS or P-GPS assistan
+
+   You can perform the reformatting and renaming as follows:
+
+   . tabs::
+
+      .. group-tab:: Bash
+
+         .. code-block:: console
+
+           awk 'NF { print "\""$0"\\n\""}' "cred_<device_uuid>_crt.pem" > client-cert.pem
+           awk 'NF { print "\""$0"\\n\""}' "cred_<device_uuid>_prv.pem" > private-key.pem
+           awk 'NF { print "\""$0"\\n\""}' AmazonrootCA1.pem > ca-cert.pem
+
+      Where ``<device_uuid>`` is replaced with the device ID you created.
+
+      .. group-tab:: PowerShell
+
+         .. code-block:: console
+            gc ".\cred_<device_uuid>_crt.pem" | %{"""$_\n"""} | out-file -encoding utf8 client-cert.pem
+            gc ".\cred_<device_uuid>_prv.pem" | %{"""$_\n"""} | out-file -encoding utf8 private-key.pem
+            gc .\AmazonrootCA1.pem | %{"""$_\n"""} | out-file -encoding utf8 ca-cert.pem
+
+      Where ``<device_uuid>`` is replaced with the device ID you created.
+
+   You should now have the following three files:
+
+     * :file:`client-cert.pem`
+     * :file:`private-key.pem`
+     * :file:`ca-cert.pem`
+
+   And the contents of each file should be similar to the following:
+
+   .. code-block:: C
+
+      "-----BEGIN CERTIFICATE-----\n"
+      "sCC8AtbNQhzbp4y01FEzXaf5Fko3Qdq0o5LbuNpVA7S6AKAkjt17QzKJAiGWHakh\n"
+      "RnwzoA2dF4wR0rMP5vR6dqBblaGAA5hN7GE2vPBHTDNZGJ6tZ9dnO6446dg9gGds\n"
+      "eeadE1HdVnUj8nb+7CGm39vJ4fuNk9vogH0nMdxjCnXAinoOMRx8EklQsR747+Gz\n"
+      "sxcdVYuNEb/E2vWBHTDNZGJ6tZC1JC9d6/RC3Vb1JC4tWnK9mk/Jw984ZuYugpMc\n"
+      "1t9umoGFYCz0nMdxjCnXAbnoOMC5A0RxcWPzxfC5A0RH+j+mwoMxwhgfFY4EhVxp\n"
+      "oCC8labNQhzRC3Vc1JC4tWnK9mpVA7k/o5LbuNpVA7S6AKAkjt17QzKJAiGWHakh\n"
+      "RXwcoAndF4wPzxfC5A0RHponmwBHTDoM7GE2vPBHTDNZGJ6tZ9dnO6446dg9gGds\n"
+      "eefdE1HcVnULbuNpVA7S6AKAkjxjCnt1gH0nMdxjCnXAinoOMRx8EklQsR747+Fz\n"
+      "srm/VYaNEb/E2vPBHTDNZGJ6tZc1JC9d6/RC3Vc1JC4tWnK9mk/Jr984ZuYugpMc\n"
+      "nt9uZTGFYCzZD0FFAA5NAC4i1PARStFycWPzxfC5A0RqodhswoMxwhgfFY4EhVx=\n"
+      "-----END CERTIFICATE-----\n"
+
+   Do not attempt to use this example directly, it has been filled with dummy data.
+
+#. Create a :file:`certs` folder directly in the :file:`nrf_cloud_mqtt_multi_service` folder, and copy :file:`client-cert.pem`, :file:`private-key.pem` and :file:`ca-cert.pem` into it.
+
+   Be sure not to place the new folder in the :file:`nrf_cloud_mqtt_multi_service/src` folder by accident.
+
+   Now these certs are automatically used if the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option is enabled.
+
+#. Build the sample with the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option enabled and the device ID you created configured
+
+   For provisioning to succeed, you must build the sample for each device, configuring each build with the corresponding device ID.
```

Comment:
```suggestion
#. Build the sample with the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` Kconfig option enabled and the device ID you created configured.
   This is required for provisioning to succeed.
```
The steps are already for each device, so no need to reiterate that, and the second sentence was mostly restating the first without that bit.